src/libponyc/codegen/genprim.c

```diff
@@ -485,7 +485,7 @@ void genprim_pointer_methods(compile_t* c, reach_type_t* t)
   pointer_lt(c, t);
```

Comment:
changes in genprim.c are unrelated to RFC. can you revert?

Reply:
sorry, correction:

`genprim_maybe_methods` is related. the others aren't.

i think.

let me look into that more.